test-skripsi/test-prod.tsx

```diff
@@ -1,5 +1,12 @@
+class InventoryError extends Error {
+	constructor(message: string) {
+		super(message);
+		this.name = "InventoryError";
+	}
```

Comment:
Consider adding more detailed documentation for the `InventoryError` class to explain its specific use cases within the inventory system.